engines/tika/src/main/java/org/alfresco/transform/tika/metadata/AbstractTikaMetadataExtractorEmbeddor.java

```diff
@@ -97,9 +97,9 @@ public abstract class AbstractTikaMetadataExtractor extends AbstractMetadataExtr
     private final DateTimeFormatter tikaUTCDateFormater;
     private final DateTimeFormatter tikaDateFormater;
 
-    public AbstractTikaMetadataExtractor(Logger logger)
+    public AbstractTikaMetadataExtractorEmbeddor(Type type, Logger logger)
```

Comment:
Abstract constructors should be `protected`

Reply:
done